src/webportal/src/app/job/job-view/fabric/job-detail/conn.js

```diff
@@ -86,8 +86,16 @@ export async function fetchSshInfo() {
   }
 }
 
-export function getJobMetricsUrl() {
-  return `${config.grafanaUri}/dashboard/db/joblevelmetrics?var-job=${namespace ? `${namespace}~${jobName}`: jobName}`;
+export function getJobMetricsUrl(getHumanizedJobStateString, jobInfo) {
```

Comment:
You can import `getHumanizedJobStateString` directly instead of passing the util function as a param

Reply:
In the summary component getHumanizedJobStateString directly call this function

Reply:
I deleted getHumanizedJobStateString function, use the jobInfo jobStatus. State status value as the judgment condition, effect